test/EstateRegistry.js

```diff
@@ -481,4 +490,20 @@ contract('EstateRegistry', accounts => {
       await assertLandIdAtIndex(estateId, 4, 5)
     })
   })
+
+  describe('land update', function() {
+    it('should not allow old owner to update land after creating an Estate', async function() {
```

Comment:
Why is this check necessary? Wouldn't it fail always as the new owner is the EstateRegistry? I mean, this check should be covered by the LANDRegistry not allowing updates by no other than the authorized+owner.

Reply:
As we are using an internal function `_doTransferFrom` on` createEstate` which cleans the `updateOperator` for the token being transferred I thought it was useful to have that test there. Once we implement the `updateLandData` on the EstateRegistry we will have a good set of tests for this subject